messageparser.go

```diff
@@ -473,6 +494,7 @@ func parseCiscoAVPair(logMessage *LogMessage, key string, value string) error {
 
 	ciscoAVPairSlice := strings.Split(value, "=")
 	if len(ciscoAVPairSlice) < 2 {
+		logMessage.CiscoAVPair = nil
```

Comment:
I think this change will cause an unintended problem, since `parseCiscoAVPair` will likely run multiple times for the same message. Essentially, each time it runs, it populates a new field in the CiscoAVPair struct, but if we set the CiscoAVPair struct pointer to nil in the case one of the fields is malformed, it would throw away any other valid data we may have already populated.